### PR TITLE
Use USWDS grid to match front-end specs; mobile improvements

### PR DESCRIPTION
**Pull request by alexsoble**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/168)

## What does this change?

+ Use the `grid-row-`, `grid-col-`, and `grid-offset-` classes from USWDS rather than our not-too-flexible, not-too-idiomatic `content-wrapper` class.
+ Match grid styles as specified by front-end specs.
+ Mobile style fix.

## What's NOT fixed by this PR? 

+ The width of the Primary Issue page radio button areas isn't affected by this PR. 
  + To make them wider, we'll need to adjust the width of the form, either on the Primary Issue page specifically or on all pages.

## Desktop screenshots

Using the USWDS grid system and matching the specs makes the form a bit narrower than on the current dev site:

### Dev site
crt-portal-django-dev app cloud gov_report_

### This PR 
0 0 0 0_8000_report_

## Mobile fixes

I added a `@media` query to make text input fields display better on small, small phones like iPhone 5s. Screenshots and more explanation in the comment below: https://github.com/usdoj-crt/crt-portal/pull/177#issuecomment-559826533.

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### CODE REVIEW

**Comment by alexsoble** [comment]
☝️ After applying `.grid-container` correctly, the layout is looking good on desktop to me! A desk check with the designers will confirm. Finding that the width of the primary-issue radio button areas will need to be a follow-on PR tweaking the max form width.

Mobile styles using the USWDS grid still need work before this PR is ready for code review.

**Comment by alexsoble** [comment]
In 703d1d83ddff5ea40b08853e7654c9e51ba6d6c9, I made a tweak for text input fields to display better on small phones (320px wide) like iPhone 5s. The input fields no longer stretch to the edge of the screen and now match the "Next" button in width. 

### Dev site

crt-portal-django-dev app cloud gov_report_(iPhone 5_SE)

### This PR

0 0 0 0_8000_report_(iPhone 5_SE)

File: crt_portal/static/sass/custom/form.scss [hunk 1]
```diff
@@ -49,7 +49,9 @@ form#report-form {
   }
 
   .usa-input {
-    width: 320px;
+    @media screen and (min-width: $tablet) {
```
Comment (alexsoble) [inline crt_portal/static/sass/custom/form.scss]: Small mobile fix here.

File: crt_portal/static/sass/custom/variables.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Variables
```
Comment (alexsoble) [inline crt_portal/static/sass/custom/variables.scss]: This file already held one non-color variable; renaming it for accuracy.

**Comment by alexsoble** [comment]
A front-end code review on this PR would be welcome come Monday when folks are back.

An Internet Explorer check, either locally or against the dev site once this merges, would be good. 

The USWDS grid system uses flexbox and we've seen that flexbox and IE11 don't always play nice, though USWDS supports IE 11: 

https://designsystem.digital.gov/documentation/developers/#browser-support

Also planning to desk check this PR with the designers during our next front-end dev/design sync. That could be showing local work or work on dev, either seems fine to me.

File: crt_portal/static/sass/custom/form.scss [hunk 3]
```diff
@@ -6,7 +6,7 @@
 form#report-form {
   .question_primary_complaint {
     ul {
-      padding-inline-start: 0;
+      padding-left: 0;
```
Comment (el-mapache) [inline crt_portal/static/sass/custom/form.scss]: `padding-inline` is super bleeding edge and has no support in any version of IE, so I replaced it with `padding-left`

**Comment by el-mapache** [comment]
@alexsoble Merging now, I did see one additional error in IE: the `legend` tag overflows its parent element on step 4.

Screen Shot 2019-12-02 at 7 45 41 AM

It's outside the scope of this issue / PR, so I'll open a new ticket to fix it.

**Comment by alexsoble** [comment]
@el-mapache Thanks for the review, the fix, and the careful IE check!

**Pull request merged**